plugins/generic/pln/templates/ping.tpl

```diff
@@ -0,0 +1,20 @@
+
+<plnplugin>
+    <ojsInfo>
+        <release>{$ojsVersion|escape}</release>
+    </ojsInfo>
+    <pluginInfo>
+        <release>{$pluginVersion.release}</release>
+        <releaseDate>{$pluginVersion.date}</releaseDate>
+        <installed>{$pluginVersion.version->getDateInstalled()}</installed>
+        <current>{$pluginVersion.version->getCurrent()}</current>
+    </pluginInfo>
+    <journalInfo>
+        <title>{$journal->getLocalizedTitle()}</title>
```

Comment:
(The journal title is dropped into your XML, and it's possible that someone will add special characters into the title -- think < or >. That'll cause the XML to fail validation. Passing it through the |escape Smarty modifier will take care of this.)